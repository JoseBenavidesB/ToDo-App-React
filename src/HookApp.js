
import React from 'react'

export const HookApp = () => {
  return (
    <h1>Hello world</h1>
  )
}
