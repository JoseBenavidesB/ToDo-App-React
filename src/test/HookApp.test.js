import React from 'react'
import { shallow } from "enzyme";


import { HookApp } from "../HookApp";

describe( 'Test <HookApp />', () => {
    
    test( "Should Return <HookApp /> correctly", () => {
        const wrapper = shallow( <HookApp/> );

        expect( wrapper ).toMatchSnapshot();
    })
});
