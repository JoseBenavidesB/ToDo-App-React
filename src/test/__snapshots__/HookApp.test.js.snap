// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <HookApp /> Should Return <HookApp /> correctly 1`] = `
<h1>
  Hello world
</h1>
`;
